components/Header/HomeButton.tsx

```diff
@@ -1,29 +1,13 @@
-import { useFeedProvider } from '@/providers/FeedProvider';
-import { FeedType } from '@/types/Feed';
-import { useRouter } from 'next/navigation';
-
-const HomeButton = () => {
-  const { push } = useRouter();
-  const { setFeedType } = useFeedProvider();
-
-  const onClick = () => {
-    setFeedType(FeedType.Trending);
-    push('/');
-  };
+import { cn } from '@/lib/utils';
+import { ChevronLeft } from 'lucide-react';
+import Link from 'next/link';
 
+const HomeButton = ({ className}: { className?: string }) => {
   return (
-    <button className="flex w-full items-center gap-3" type="button" onClick={onClick}>
-      <svg width="8" height="14" viewBox="0 0 8 14" fill="none" xmlns="http://www.w3.org/2000/svg">
-        <path
-          d="M7 13L1 7L7 1"
-          stroke="#333536"
-          strokeWidth="2"
-          strokeLinecap="round"
-          strokeLinejoin="round"
-        />
-      </svg>
+    <Link href='/' className={cn("flex items-center gap-1", className)}>
+      <ChevronLeft/>
       <p className="font-sora text-[18px] font-semibold">Home</p>
-    </button>
+    </Link>
   );
 };
```

Comment:
return is redundant right?
- remove `return` 🚮 
- remove curly braces `{}` 🚮 

same functionality with less code right?